### DISCUSSION TITLE
gatsby-plugin-image styles on <head>

### DISCUSSION THREAD

**Discussion by Lange1990** [gatsbyjs/gatsby#31974] [Help]
I noticed that default styles of gatsby-plugin-image are been loading at <head> like <style>, is that normal?

<style>.gatsby-image-wrapper{position:relative;overflow:hidden}.gatsby-image-wrapper img{bottom:0;height:100%;left:0;margin:0;max-width:none;padding:0;position:absolute;right:0;top:0;width:100%;object-fit:cover}.gatsby-image-wrapper [data-main-image]{opacity:0;transform:translateZ(0);transition:opacity .25s linear;will-change:opacity}.gatsby-image-wrapper-constrained{display:inline-block;vertical-align:top}</style>

The problem is that differs the order when you are in Develop or in Production mode.
On Develop my own styles are down and the plugin styles are up, so if for example i put one className with position : absolute this one will step the position: relative of  gatsby-image-wrapper. 
But on Production the css file from the bundle its at the top of the head, and <style> of the plugin its at the bottom. This cause that what you see on :8000 its not the same on :9000.

Im using miniCssExtract so in Production i allways have the css file at the top of <head>.

This i can fix it by putting inline style, or saying in className that is !importan on scss.

Is there any way to avoid that <style> element on the html? 
Why is the only plugin that generates that <style> element on the html?
Does gatsby have one way that i can say put the css file at the bottom of <head> ?

Im using:
- "gatsby": "^3.6.2"
- "gatsby-plugin-image": "^1.6.0"
- "sass": "^1.34.1"
- "gatsby-plugin-sass": "4.7.1"

**Comment by LekoArts** [comment]
Hi!

It's working in intended when it's adding the style to the head. You can use `style` or `imgStyle` to override some styles (https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#shared-props) or you use `!important` as you mentioned.

**Reply by Lange1990** [reply]
Thanks , i wasnt shure if was normal because with gatsby-image didnt happen this. 
Thanks!!!
